services/datamanager/builtin/builtin.go

```diff
@@ -48,7 +48,7 @@ func init() {
 			},
 			// NOTE(erd): this would be better as a weak dependencies returned through a more
 			// typed validate or different system.
-			WeakDependencies: []internal.ResourceMatcher{internal.ComponentDependencyWildcardMatcher},
+			WeakDependencies: []internal.ResourceMatcher{internal.ComponentDependencyWildcardMatcher, internal.SLAMDependencyWildcardMatcher},
```

Comment:
Q for my understanding: what is this?

Reply:
Will talk about more in our eng meeting, but the abridged version is `WeakDependencies` are used within in manager so we can implicitly capture methods without having to specify `depends_on`

Reply:
Thanks for following up with Eric about this.